src/modules/user/domain/models/user.model.ts

```diff
@@ -35,6 +37,10 @@ export class UserModel extends AggregateRoot {
     return this._entityRoot.status.value;
   }
 
+  get archivedAt(): Date | undefined {
+    return this._entityRoot.archivedAt._value;
```

Comment:
Hay que agregar una validacion a esto porque puede que este undefined y saldra el tipico error de que can not read properties of undefined

Reply:
@2314-RAustin Esto también aplica para el modelo de role?

Reply:
Si, cambiarlo en role también por favor

Reply:
@2314-RAustin Listo, corregido.